dear_petition/petition/export/forms.py

```diff
@@ -138,6 +140,22 @@ def map_offenses(self):
             self.data[f"DOOF:{i}"] = self.format_date(ciprs_record.offense_date)
             self.data[f"DismissalDate:{i}"] = self.format_date(offense.disposed_on)
 
+    def map_same_day_offenses(self):
+        dismissed_records = self.petition.get_all_offense_records()
+        charges = dismissed.same_day_convictions(dismissed_records)
+        line = []
+        for charge in charges[:1]:
+            line.append(
+                f"{charge.offense.ciprs_record.file_no} {charge.description.title()}"
+            )
+        self.data["ChargedDesc"] = " ".join(line)
+        line = []
+        for charge in charges[1:]:
+            line.append(
+                f"{charge.offense.ciprs_record.file_no} {charge.description.title()}"
+            )
+        self.data["ChargedDescCont"] = " ".join(line)
```

Comment:
I am a bit confused by what all of this is doing. What is ChargedDesc and ChargedDescCont? I don't see those fields anywhere. I'm also not seeing the Checkbox annotation being used anywhere. Did the description of the issue change?